src/omero_cli_render.py

```diff
@@ -414,7 +420,7 @@ def _lookup(self, gateway, type, oid):
             self.ctx.die(110, "No such %s: %s" % (type, oid))
         return obj
 
-    def render_images(self, gateway, object, batch=100):
+    def load_images(self, gateway, object, batch=100):
```

Comment:
Can you explain your thoughts on this rename?

Reply:
Because the method doesn't render any images, it just loads the images. I found the name `render_images` totally confusing every time I looked into the omero_cli_render code.